core/adapters/random_test.go

```diff
@@ -0,0 +1,44 @@
+package adapters_test
+
+import (
+	"math/big"
+	"testing"
+
+	"github.com/smartcontractkit/chainlink/core/adapters"
+	"github.com/smartcontractkit/chainlink/core/store/models"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestRandom_Perform(t *testing.T) {
+	tests := []struct {
+		name    string
+		errored bool
+	}{
+		{"resulting string parses to a big.Int 1", false},
+		{"resulting string parses to a big.Int 2", false},
+		{"resulting string parses to a big.Int 3", false},
+		{"resulting string parses to a big.Int 4", false},
+		{"resulting string parses to a big.Int 5", false},
```

Comment:
Are these test cases just running the same test 5 times? I think we can just collapse it to one.